content/en/docs/advanced/edgemesh.md

```diff
@@ -145,7 +145,7 @@ To request a server, use url like this: `<service_name>.<service_namespace>.svc.
 
 In our case, from edge node a or b, run following command:
 
-```bash
+```html
```

Comment:
Same as above.

Reply:
I guess separating the command and result by putting them in different code block would work!

Reply:
Yes. That would be better.